packages/lisk-p2p/src/peer_pool.ts

```diff
@@ -142,7 +146,7 @@ export class PeerPool extends EventEmitter {
 		return selectedPeers;
 	}
 
-	public async requestPeer(
+	public async requestFromPeer(
```

Comment:
why change the name to `requestFromPeer`? `requestPeer` sounds more straight forward?

Reply:
- The other method is called 'sendToPeer', so 'requestFromPeer' sounds more consistent.

- requestPeer is a bit confusing because someone who doesn't know the code could expect the method to return a Peer object.

I guess the second point is not a big deal with TypeScript though but I think we should. be consistent with the naming. If we call it 'requestPeer' then maybe we should call the other method 'sendPeer'?